erizo/src/erizo/WebRtcConnection.h

```diff
@@ -114,11 +108,6 @@ class WebRtcConnection: public TransportListener, public LogContext, public Hand
    * Copy some SdpInfo data to local SdpInfo
    */
   void copyDataToLocalSdpIndo(std::shared_ptr<SdpInfo> sdp_info);
```

Comment:
I know this is not from this PR but it may make sense to change this to `copyDataToLocalSdpInfo` as part of the cleaning

Reply:
good catch!